src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,71 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
 
+        Scanner scanner = new Scanner(System.in);
+        makeChoice();
+        String input;
+        int a, b;
+        while (true) {
+            while (!scanner.hasNext("(exit|help|add|sub|mul|div)")) {
+                System.out.println("False, please try again.");
+                makeChoice();
+                scanner.next();
+            }
+            input = scanner.next();
+
+            if (input.equals("exit")) {
+                System.out.println("Bye-bye");
+                scanner.close();
+                break;
+            }
+            if (input.equals("help")) {
+                makeChoice();
+                scanner.next();
+            }
```

Comment:
I personally don't like such solution, where you move some `switch` clauses into separate `if-else` statements, I'd really prefer some other implementation where all possible commands would be inside one `switch`.